frontend/index.html

```diff
@@ -1,12 +1,43 @@
 <!doctype html>
 <html lang="fr">
+
 <head>
   <meta charset="utf-8">
   <title>Dreamworks</title>
+  <meta name="description" content="Data visualisation practice project @Telecom ParisTech.">
+  <meta name="author" content="Alexis Bauvin, Ariane Robineau, Clément Decoodt, Jean Lepeudry, Léa Saviot">
   <link rel="stylesheet" href="css/index.css">
-  <script src="js/index.js"></script>
+
+  <!-- Bootstrap -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+  <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>
+  <!-- End Bootstrap -->
+
+  
 </head>
+
 <body>
-  <!-- put content here -->
+	<div id="pagetitle">
+		<h1>Dreamworks • Visualization of a dream bank</h1>
```

Comment:
dot char is dangerous, see [this site](https://www.sitepoint.com/web-foundations/use-unicode-create-bullet-points-trademarks-arrows/) for more info :)